docker-compose-test.yml

```diff
@@ -54,6 +54,22 @@ services:
       - "9000:9000"
     command: start
 
+  # ---------------------------------
```

Comment:
I'm wondering if this must go to `./aether-utils/docker-compose.yml` file. Something like this:

```yaml
  # ---------------------------------
  # Aether Kernel Client TEST dependencies
  # ---------------------------------
  db-test:
    extends:
      file: ../docker-compose-test.yml
      service: db-test

  kernel-test:
    extends:
      file: ../docker-compose-test.yml
      service: kernel-test

  # ---------------------------------
  # Aether Kernel Client
  # ---------------------------------
   client-test:
    build: ./aether-client
    env_file:
      - ../../.env
    environment:
      # use a different database when testing the integration with another container
      KERNEL_URL: http://kernel-test:9000
      KERNEL_ADMIN_USERNAME: ${KERNEL_ADMIN_USERNAME}
      KERNEL_ADMIN_PASSWORD: ${KERNEL_ADMIN_PASSWORD}
      TESTING: "true"
      VERSION: ${VERSION}
    volumes:
      - ./aether-client:/code
    command: test
    depends_on:
      db-test:
        condition: service_healthy
      kernel-test:
        condition: service_healthy
```